src/accounting.py

```diff
@@ -906,6 +914,9 @@ def _read_ledger(self, ledger_path):
                 self.get_tax_object().toggle_auto_tax()
             elif cmd == 'force-tax':
                 pass
+            elif cmd == 'mark-public':
+                super().mark_public(parse_account_id(elems[1]), self.get_account_from_string(elems[2]), True if elems[3] == "True" else False)
```

Comment:
I think you can rewrite `True if elems[3] == "True" else False` as `elems[3] == "True"`.